### ISSUE TITLE
Additions to species list for Agromyzidae Recording Scheme

### ISSUE THREAD

**Issue by DavidRoy** [BiologicalRecordsCentre/iRecord#640]
To be added to taxon lists to enable recording in iRecord.  Adding to August 2019 milestone as import for supporting active Recording Scheme on iRecord.

Agromyza abdita 
Agromyza macedonica
Cerodontha palustris
Cerodontha pygmella *
Cerodontha zuskai *
Liriomyza latigenis 
Liriomyza obliqua 
Liriomyza taurica *
Liriomyza yasumatsui 
Melanagromyza galegae 
Phytomyza phillyreae 
Phytomyza penicilla
* = species not yet published as new to Britain but paper in print.

**Comment by johnvanbreda** [comment]
Names can presumably be added via the UI. I think @kitenetter already has notes - the key thing is to set the TVK into both the external key and search code fields so it maps properly to UKSI.

**Comment by burkmarr** [comment]
@DavidRoy - are any of these likely to be in the latest UKSI? In other words, should I tackle this after the UKSI is updated?

**Comment by DavidRoy** [comment]
I think these are ideally tackled before the UKSI update, as it should be a reasonable simple upload.  @kitenetter to advise

**Comment by burkmarr** [comment]
Have you got time for a short telephone conversation on this @kitenetter?

**Comment by kitenetter** [comment]
@burkmarr I'm now back in the office and wading through emails, happy to discuss this via phone call whenever is good for you. If these names are already in UKSI then it is reasonably straightforward to add them to iRecord and I'm happy to do that or for you to. But first job is to check if they are in UKSI, which I would normally do by searching at http://nbn-sd-dev.nhm.ac.uk/index.php

**Comment by kitenetter** [comment]
In fact only one of those new names is in the UKSI so the first step is to get Chris to add them to UKSI. But he may not be able to do that for the names that are not yet published, and will definitely need info on the authority for each name. I will liaise with Barry for the recording scheme.

**Comment by burkmarr** [comment]
Linked to #636

**Comment by burkmarr** [comment]
Once problems with UKSI update are sorted (#636) @burkmarr to touch base with @kitenetter to see if UKSI has been updated with these taxa and, if so, request new UKSI update (Access DB) from Chris before updated the main warehouse UKSI taxon list. If the taxa have not been included by that point we will proceed with the UKSI update and then add these taxa in manually via the Warehouse once Chris has assigned keys (i.e. added to UKSI).

**Comment by kitenetter** [comment]
Chris Raper added the new names to UKSI today, and I've now imported them via the csv import into the warehouse. The import process is documented here:
W:\PYWELL_SHARED\Pywell Projects\BRC\_BRC_projects\iRecord\Procedures and guidance\PROCEDURE adding new species to the iRecord dictionary.docx

Some questions remain over how to deal with English names and synonyms, should they be required (which they weren't in this case) - I will raise these as separate issues when needed.

I will also copy the template to Chris to see if it will be suitable for the more regular batch updates that he proposes we use in future (following the full update under #636).

Closing.
